### DISCUSSION TITLE
Allow to disable auto scroll

### DISCUSSION THREAD

**Discussion by m-j-g** [flyerhq/flutter_chat_ui#317] [Ideas & Requests]
I get a lot of complaints about it being impossible to catch up in a group chat because they scroll up to previous messages and then people keep sending messages and it auto scrolls them down to the newest message.  I'm trying to figure out a way to only scroll to the newest message when the user is already scrolled all the way down, but if they have any scroll inset then I want to not scroll to the newest message.  

I tried my best to look through the methods on the Chat() object and couldn't find a solution.  Has anyone been able to accomplish this?

**Comment by demchenkoalex** [comment]
It should not autoscroll to every message, it should only autoscroll to the message you send, e.g. if I send something I see a scroll, if you send something or whatever person - it should not scroll. If this happens - it is a bug.

**Comment by m-j-g** [comment]
Oh man, you mean this whole time it was because I was doing something goofy!?  I probably just need to look at how I'm managing/overwriting messages when new ones come in.  Thanks for letting me know that it was possible!
